src/content/api/promotion-memberships.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Promotion Memberships
+description: Promotion membership model and related endpoints
+nav:
+  path: API/Loyalty Programs
+  order: 4
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+import Error from '../../components/Error.astro';
+
+Promotion Memberships represents an Account's membership to a Promotion. Each Promotion Membership carries progress towards completing the Promotion. Promotion Memberships are created automatically for the account when a Loyalty Program Membership is created.
```

Comment:
Nice description :) you could include links to models for Account, Promotion, and Loyalty Program Membership

Reply:
Yep, good idea :)